tests/tests_e3/job/scheduling_test.py

```diff
@@ -90,6 +92,9 @@ def collect(job):
             assert v[0] is None
             assert v[1] is None
 
+    @pytest.mark.xfail(sys.platform.startswith('sunos'),
```

Comment:
I would prefer a test:

try:
    import psutil
except ImportError:
   psutil = None

@pytest.mark.xfail(psutil is None, ...):

Also fix your commit message ;)